app/editor/src/features/admin/users/UserFilter.tsx

```diff
@@ -1,14 +1,27 @@
-import { IconButton } from 'components/form';
-import React from 'react';
+import { IconButton, OptionItem, Select } from 'components/form';
+import { UserStatus, UserStatusName } from 'hooks';
+import React, { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import { Row, Text } from 'tno-core';
+import { useUsers } from 'store/hooks/admin';
+import { FieldSize, Text } from 'tno-core';
+import { Row } from 'tno-core/dist/components/flex';
 
-interface IUserFilterProps {
-  setGlobalFilter: (filterValue: any) => void;
-}
+import { IUserListFilter } from './interfaces/IUserListFilter';
+
+interface IUserFilterProps {}
+
+export const UserFilter: React.FC<IUserFilterProps> = () => {
+  const [{ userFilter }, { storeFilter }] = useUsers();
+  const [filter, setFilter] = useState<IUserListFilter>(userFilter);
+  const statusOptions = [
```

Comment:
@jdtoombs You're mixing enums up here I believe.  The API will convert enum values into string values.  Can you please confirm what the API is doing?

Reply:
Looks like it is sending the numeric value to the API, it seems to be filtering correctly too

Reply:
While the API may understand how to deserialize the number to an enum, it is configured to serialize enum values as strings.  Which means you should be able to just use the string values.  Can you test the string values and see if it works?  It'll make you code a lot cleaner and the filter will be more human readable.

Reply:
I tried passing it just the string values; however, the `Select` component will throw an error as it is expecting `OptionItems` which require more than just a string value. Is this something I should try to tweak? I noticed the other areas in which we use the Select component they follow the same format as I am doing above

Reply:
Do we do this exact type of filter with content?

Reply:
Oh wait maybe I am misunderstanding, I took this as you wanting to pass ["item 1", "item 2"] but are you just wanting the value sent to the api to be the string? So it could be OptionItem("label", stringValue)?

Reply:
The filter sent to the API should be the string value by itself "value".  Which the API will understand and deserialize into the appropriate enum.

Reply:
Think I may have it now, I was overcomplicating it in my head thinking you just wanted to provide the label, with no value.. which obviously didn't click for me at first lol

Reply:
Won't `UserStatusName.Preapproved === 'Preapproved'`?  Why do you need to manually add the string values?  Don't we have an existing method that converts enums to options?